configs/next/packages/valgrind/sources

```diff
@@ -51,6 +51,11 @@ atsrc_get_patches ()
 	at_get_patch \
 		'https://raw.githubusercontent.com/powertechpreview/powertechpreview/master/Valgrind%20iTrace%20Patches/3.14/update_itrace_to_use_epoch.v2.diff' \
 		41fc03e4b074698f1fff9e7a53fed63a || return ${?}
+
+	# Update itrace to Valgrind 3.15.
+	at_get_patch \
+		'https://raw.githubusercontent.com/powertechpreview/powertechpreview/master/Valgrind%20iTrace%20Patches/3.15/Fix-itrace-initialization-of-guest_chase-variable.patch' \
```

Comment:
I suggest to use https://raw.githubusercontent.com/powertechpreview/powertechpreview/0496d9de5cf3d441787d413367c5ec514a64b2ae/Valgrind%20iTrace%20Patches/3.15/Fix-itrace-initialization-of-guest_chase-variable.patch instead, it's better to use a fix commit ID in case that file is updated and breaks the build.

Reply:
Good point. Thx.